src/Exception/ExtensionRequirementException.php

```diff
@@ -27,6 +26,12 @@ class ExtensionRequirementException extends RuntimeException implements GoogleLi
 	public static function missing_required_plugin( string $plugin_name ): ExtensionRequirementException {
 		return new static(
 			sprintf(
+				'Requires %1$s to be enabled.', // Fallback exception message.
```

Comment:
I may miss something, but why don't we include the plugin name in the fallback message? 

I get that it cannot be translated, but as we hardcode the message in English, we could also hardcode the plugin name in English.

Reply:
I was a bit on the fence with the fallbacks. Initially I just used a string like `missing_required_plugin` but thought we might as well fallback to some English text. We wouldn't expect to ever see the fallback so generally it shouldn't make a difference. I just thought it would be less work if we renamed the extension again. But I'll add them back in to make the messages similar.